.github/workflows/test.yml

```
@@ -15,11 +15,130 @@ jobs:
+      - name: '[Run] Negative test: Invalid image name'
+        id: invalid-image
+        if: always() && steps.pre-setup.outcome == 'success'
+        continue-on-error: true
+        uses: ./
+        with:
+          image: invalid/image:latest
+
+      - name: '[Verification] Negative test: Invalid image name'
+        if: always() && steps.pre-setup.outcome == 'success'
+        shell: bash
+        run: |
```

actionlint (1.7.4)

property "error" is not defined in object type {}

(expression)